src/logcollector/logcollector.c

```diff
@@ -1138,6 +1138,20 @@ int check_pattern_expand(int do_seek) {
                     mwarn(FILE_LIMIT, maximum_files);
                     break;
                 }
+
+                struct stat statbuf;
+                if (lstat(g.gl_pathv[glob_offset], &statbuf) < 0) {
+                    merror("Error on stat '%s' due to [(%d)-(%s)]", g.gl_pathv[glob_offset], errno, strerror(errno));
+                    glob_offset++;
+                    continue;
+                }
+
+                if ((statbuf.st_mode & S_IFMT) != S_IFREG) {
+                    mdebug2("Ignoring '%s'. It's not a regular file", g.gl_pathv[glob_offset]);
```

Comment:
Change it to debug1, the same for the Windows log at https://github.com/wazuh/wazuh/blob/315f2e1fa7963c55c077591ea1e6666f670c5566/src/logcollector/logcollector.c#L1344.

Use the same format as the Windows existing log for this one: `mdebug1("File %s is not a regular file. Skipping it.", g.gl_pathv[glob_offset]);`